frontend/talentsearch/src/js/components/search/SearchPools.tsx

```diff
@@ -64,7 +66,11 @@ const SearchPools: React.FunctionComponent<SearchPoolsProps> = ({
         )}
       </p>
       <p data-h2-margin="base(x1, 0)">{pool?.description?.[locale]}</p>
-      <Button color="cta" mode="solid" onClick={handleSubmit}>
+      <Button
+        color="cta"
+        mode="solid"
+        onClick={() => handleSubmit(candidateCount)}
```

Comment:
I think this is requesting all the pools on the page, not just the one that I clicked.  Here's what it looks like when there were two pools.
![image](https://user-images.githubusercontent.com/8978655/199103658-0db045b4-7052-45ca-9814-1b6579b69f54.png)
![image](https://user-images.githubusercontent.com/8978655/199104147-83598ede-a5f6-4f6d-8035-88a10a65ca40.png)

Reply:
I believe this should fix it [48bc7c7](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4528/commits/48bc7c7c56653a1740819bf13371cf4241bca8e2). Nice catch 👍